LayoutTests/imported/w3c/web-platform-tests/css/css-animations/display-none-dont-cancel.tentative.html

```diff
@@ -72,7 +76,8 @@
 }
 </style>
 <script>
-promise_test(async () => {
+promise_test(async (t) => {
+  t.add_cleanup(() => target3.classList.remove('animate3'));
```

Comment:
Nit: you added a blank line after this in the previous subtests. Let's be consistent about it in this and the later subtests.

Reply:
Also: do we still need to make these changes in this test, if the new test you're adding is in a separate file?

Reply:
It's nice to cleanup after tests either way, though not needed.